### ISSUE TITLE
Add option for filtering commits by category

### ISSUE THREAD

**Issue by gamichalski** [lob/generate-changelog#39]
Its amazing!!

It would be awesome if I could filter and exclude them by category

Would you accept a PR?

**Comment by robinjoseph08** [comment]
Hey @gamichalski,

Just to clarify, do you mean to have a flag similar to `--exclude` but instead of excluding types during generation, excluding categories as well?

**Comment by gamichalski** [comment]
Yes that's right.
Is it my need, is it good for everyone?

**Comment by robinjoseph08** [comment]
Gotcha. I'm not sure if I understand the use-case just yet. Can you give an example of your workflow that requires the filtering of a category? Is it possible to just group those under `chore` and then exclude that type?

**Comment by gamichalski** [comment]
Yes, I would like to make a separation from what is related to a product, and what is technology related

**Comment by wopian** [comment]
I'm using `lerna` for my monorepos which is another use case for this feature (built-in changelog generation of lerna sucks for non-`feat`/`fix` commits). 

The package name is being used as the category (e.g `feat(kitsu-core): message`). Exclusion by category would allow ignoring commits for `package A` in `package B`'s changelog.
